app/model/user.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const mongoose = require('mongoose');
+// const bcrypt = require('bcrypt');
+
+let userSchema = mongoose.Schema({
+  username: {type: String, required: true, unique: true},
```

Comment:
It's generally easier to break these out into their own lines, especially since more options can be set.

```javascript
username: {
  type: String,
  ...
}
```